wear/src/main/java/io/homeassistant/companion/android/HomeAssistantApplication.kt

```diff
@@ -1,7 +1,39 @@
 package io.homeassistant.companion.android
 
 import android.app.Application
+import android.content.Intent
+import android.content.IntentFilter
+import android.os.PowerManager
 import dagger.hilt.android.HiltAndroidApp
+import io.homeassistant.companion.android.sensors.SensorReceiver
 
 @HiltAndroidApp
-open class HomeAssistantApplication : Application()
+open class HomeAssistantApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        val sensorReceiver = SensorReceiver()
+        // This will cause the sensor to be updated every time the OS broadcasts that a cable was plugged/unplugged.
+        // This should be nearly instantaneous allowing automations to fire immediately when a phone is plugged
+        // in or unplugged. Updates will also be triggered when the system reports low battery and when it recovers.
+        registerReceiver(
+            sensorReceiver,
+            IntentFilter().apply {
+                addAction(Intent.ACTION_BATTERY_LOW)
+                addAction(Intent.ACTION_BATTERY_OKAY)
+                addAction(Intent.ACTION_POWER_CONNECTED)
+                addAction(Intent.ACTION_POWER_DISCONNECTED)
+            }
+        )
+
+        // This will cause interactive and power save to update upon a state change
+        registerReceiver(
+            sensorReceiver,
+            IntentFilter().apply {
+                addAction(Intent.ACTION_SCREEN_OFF)
+                addAction(Intent.ACTION_SCREEN_ON)
+                addAction(PowerManager.ACTION_POWER_SAVE_MODE_CHANGED)
+            }
+        )
+    }
```

Comment:
We probably don't need these until we add the interactive and power save sensors, otherwise we will trigger a lot of unwanted updates :)